README.md

```diff
@@ -111,6 +111,61 @@ $ sudo sh -c "echo fs.inotify.max_user_watches=524288 >> /etc/sysctl.conf"
 $ sudo sysctl -p
 ```
 
+## i18n
```

Comment:
looking at the size of the section i'd create a new file specifically for i18n documentation
in the main readme we would just do
```
See [i18n](link) for information on our internationalization tools and guidelines.
```

Reply:
Ok!